solidity-v1/dashboard/src/components/modal/threshold/WithdrawGrantedTokens.jsx

```diff
@@ -0,0 +1,195 @@
+          <form>
+            {grants
+              .slice(0, numberOfGrantsDisplayed)
+              .map((grant) => renderGrant(grant, grants.length, formik))}
+          </form>
+          <OnlyIf condition={grants.length > numberOfGrantsDisplayed}>
+            <div
+              className="withdraw-granted-tokens__view-more-container"
+              onClick={() =>
+                setNumberOfGrantsDisplayed(numberOfGrantsDisplayed + 5)
+              }
+            >
+              <span>+&nbsp;</span>
+              View More
+            </div>
+          </OnlyIf>
+        </div>
+      </ModalBody>
+      <ModalFooter>
+        <SubmitButton
+          className="btn btn-primary btn-lg mr-2"
+          type="submit"
+          onSubmitAction={(awaitingPromise) => {
+            onWithdrawClick(awaitingPromise)
+          }}
+          disabled={formik.values.grantIds.length === 0}
+        >
+          withdraw
+        </SubmitButton>
+        <Button className={`btn btn-unstyled text-link`} onClick={onClose}>
+          Cancel
+        </Button>
+      </ModalFooter>
+    </>
+  )
+})
+
+const renderGrant = (grant, totalNumberOfGrants, formik) => {
+  return (
+    <div
+      key={`Grant-${grant.id}`}
+      style={{
+        display: "flex",
+        alignItems: "baseline",
+      }}
+    >
+      <FormCheckboxBase
+        className="withdraw-granted-tokens__form_checkbox mr-1"
+        name="grantIds"
+        type="checkbox"
+        value={grant.id}
+        checked={formik.values.grantIds.includes(grant.id)}
+        onChange={(e) => {
+          const set = new Set(formik.values.grantIds)
+          if (e.target.checked) {
+            set.add(grant.id)
+          } else {
+            set.delete(grant.id)
+          }
+          formik.setFieldValue("grantIds", [...set])
+        }}
+      />
+      <Accordion
+        allowZeroExpanded={totalNumberOfGrants > 1}
+        className={"withdraw-granted-tokens__grants-accordion"}
+      >
+        <AccordionItem
+          {...(totalNumberOfGrants === 1
+            ? { dangerouslySetExpanded: true }
+            : {})}
+        >
+          <AccordionItemHeading>
+            <AccordionItemButton>
+              <div className="withdraw-granted-tokens__token-amount">
+                <TokenAmount
+                  amount={grant.readyToRelease}
+                  amountClassName={"h4 text-mint-100"}
+                  symbolClassName={"h4 text-mint-100"}
+                  token={KEEP}
+                />
+              </div>
+              <span className="withdraw-granted-tokens__details-text">
+                Details
+              </span>
+              <OnlyIf condition={totalNumberOfGrants > 1}>
+                <span className="withdraw-granted-tokens__expand-button" />
+              </OnlyIf>
+            </AccordionItemButton>
+          </AccordionItemHeading>
+          <AccordionItemPanel>
+            <div className="withdraw-granted-tokens__info-row">
```

Comment:
We can use existing classes instead of creating the new one. eg `flex row center`.

Reply:
Fixed in baf4f98